mlops/tests/test_classification_metrics.py

```diff
@@ -626,3 +626,36 @@ def test_mlops_recall_score_apis():
     assert len(recall_score) == 3
 
     pm.done()
+
+
```

Comment:
most likely IntelliJ tells you that there should be 1 line interval

Reply:
it does!